docs/reactivity.md

```diff
@@ -295,7 +295,7 @@ In the future, Plot will support more interaction methods, including brushing. P
 
 The `view` function does two things:
 
-1. it [displays](../javascript#explicit-display) the given DOM *element*, and then
+1. it [displays](<../javascript#display(value)>) the given DOM *element*, and then
```

Comment:
Why is this change necessary? Isn’t the original link valid?

https://observablehq.com/framework/javascript#explicit-display

Reply:
my bad!